src/Cache/CacheClient.php

```diff
@@ -750,6 +758,13 @@ public function setLength(string $cacheName, string $setName): SetLengthResponse
      */
     public function setRemoveElement(string $cacheName, string $setName, string $element): SetRemoveElementResponse
     {
-        return $this->dataClientWrapper->getClient()->setRemoveElement($cacheName, $setName, $element);
+        return $this->getNextDataClient()->setRemoveElement($cacheName, $setName, $element);
+    }
+
+    private function getNextDataClient(): ScsDataClient
+    {
+        $client = $this->dataClients[$this->nextDataClientIndex]->getClient();
+        $this->nextDataClientIndex = ($this->nextDataClientIndex + 1) % sizeof($this->dataClients);
```

Comment:
`sizeof` is a deprecated alias of `count`

Reply:
gtk thanks